README.md

```diff
@@ -1 +1 @@
-packages/ai/README.md
\ No newline at end of file
+packages/ai/README.md
```

Comment:
revert

Reply:
I haven't changed anything in that file.